Update Dynatrace docs with metric metadata (#10)
dynatrace-oss-contrib/micrometer-docs: src/docs/implementations/dynatrace.adoc

    api-token: YOUR_METRICS_INGEST_TOKEN
----

=== Meter metadata

> Starting with Micrometer 1.12.0, the Dynatrace v2 exporter exports meter metadata to the Dynatrace API via the https://www.dynatrace.com/support/help/extend-dynatrace/extend-metrics/reference/metric-ingestion-protocol#metadata[metrics ingestion protocol].

Reviewer: I wonder if that's too much of an implementation detail to point at in the first sentence.
Users might mistakenly think they'll find their display name and unit in `dt.meta.displayName` and `dt.meta.unit` instead of the first class properties in the UI.

Author: ```suggestion
Starting with Micrometer 1.12.0, the Dynatrace registry v2 exports meter metadata to Dynatrace.
```
Like this maybe?

> Currently supported metadata are unit (called base unit in Micrometer) and description.

Reviewer: ```suggestion
Currently supported types of metadata are *unit* (called "base unit" in Micrometer) and *description*.
```

> No changes are required to start exporting Metadata to Dynatrace - upgrading to version 1.12.0 or above is enough.

Author: ```suggestion
No changes are required to start exporting Metadata to Dynatrace - upgrading to version 1.12.0 or above is enough.
Find more information about metrics metadata in the https://www.dynatrace.com/support/help/extend-dynatrace/extend-metrics/reference/metric-ingestion-protocol#metadata[Dynatrace documentation].
```

> The export of metrics metadata can be disabled via the `exportMeterMetadata` property on the `DynatraceConfig` (see <<bookmark-available-properties, the section on available properties>> below).

Reviewer: ```suggestion
The export of metrics metadata can be disabled by setting the `exportMeterMetadata` property on the `DynatraceConfig` (see <<bookmark-available-properties, the section on available properties>> below) to `false`.
```

> The export of metrics metadata can be turned off by using the `exportMeterMetadata` property on the `DynatraceConfig` (see <<bookmark-available-properties, the section on available properties>> below).

Reviewer 2: nit

```suggestion
The export of metrics metadata can be disabled via the `exportMeterMetadata` property on the `DynatraceConfig` (see <<bookmark-available-properties, the section on available properties>> below).
```
